src/libData/AccountData/AccountStore.cpp

```diff
@@ -235,6 +238,90 @@ bool AccountStore::MoveUpdatesToDisk() {
   return true;
 }
 
+bool AccountStore::RepopulateStateTrie() {
+  LOG_MARKER();
+
+  unsigned int counter = 0;
+  bool batched_once = false;
+
+  vector<pair<Address, bytes>> entries;
+
+  for (const auto& i : m_state) {
+    counter++;
+
+    if (counter >= ACCOUNT_IO_BATCH_SIZE) {
+      // Write into db
+      if (!BlockStorage::GetBlockStorage().PutTempState(
+              *this->m_addressToAccount)) {
+        LOG_GENERAL(WARNING, "PutTempState failed");
+      } else {
+        // this->m_addressToAccount->clear();
+        entries.clear();
+        counter = 0;
+        batched_once = true;
+      }
+    }
+
+    Address address(i.first);
+
+    if (!batched_once) {
+      auto iter = this->m_addressToAccount->find(address);
+
+      if (iter != this->m_addressToAccount->end()) {
```

Comment:
Can simplify like this, to reduce the temporarily variable (iter) usage.

       if (this->m_addressToAccount->find(address) != this->m_addressToAccount->end()) {

Reply:
Got it